src/ResourceManager/Sql/AzureRM.Sql.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '2.8.0'
```

Comment:
@jaredmoo please revert this change. We will bump the version during the release